packages/cli/generators/example/index.js

```diff
@@ -13,6 +13,8 @@ const utils = require('../../lib/utils');
 
 const EXAMPLES = {
   codehub: 'A GitHub-like application we used to use to model LB4 API.',
+  'getting-started':
+    'A basic Todo application used as an introduction to LoopBack 4.',
```

Comment:
Ideally, this description should match the description in `package.json`. See https://github.com/strongloop/loopback-next/pull/869/files#diff-27ee36b5a6726e3a1fa485380948168aR4:

```
"description": "An application and tutorial on how to build with LoopBack 4.",
```